src/uint.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(clippy::many_single_char_names, clippy::suspicious_arithmetic_impl)]
```

Comment:
I guess the Rust 1.22 build doesn't like this. Putting it behind the clippy feature flag should make everything happy:

```suggestion
#![cfg_attr(
    feature = "cargo-clippy",
    allow(clippy::many_single_char_names, clippy::suspicious_arithmetic_impl)
)]
```